packages/react/src/AnchoredOverlay/AnchoredOverlay.tsx

```diff
@@ -93,6 +94,10 @@ interface AnchoredOverlayBaseProps extends Pick<OverlayProps, 'height' | 'width'
    * If true, the overlay will attempt to prevent position shifting when sitting at the top of the anchor.
    */
   pinPosition?: boolean
+  /**
+   * Optional prop to set variant for narrow screen sizes
+   */
+  variant?: ResponsiveValue<'anchored', 'anchored' | 'full-screen'>
```

Comment:
```suggestion
  variant?: ResponsiveValue<'anchored', 'anchored' | 'fullscreen'>
```
Small suggestion.. for Dialog, we use `fullscreen` and I think it would be nice to align here for simplicity 😄

Reply:
Good note! Done